servicecontrol/upgrades/3to4/index.md

```diff
@@ -0,0 +1,102 @@
+
+This split is transparent to the other components of the Particular Software Platform, which should continue to connect to the main ServiceControl instance. All queries to the main ServiceControl instance will contain results from the Audit instance as well.
+
+When upgrading a ServiceControl instance to version 4, if it is configured to manage an audit queue, a new ServiceControl Audit instance will be created as a part of the upgrade process. A user will need to supply additional information about the new ServiceControl Audit instance.
+
+<Details on what that looks like in SMCU>
+
+NOTE: If the ServiceControl instance being upgraded is not configured to manage an audit queue (by setting the audit queue name to `!disable`), then no new ServiceControl Audit instance will be created.
+
+### Upgrading with PowerShell
+
+Use the `Invoke-ServiceControlInstanceUpgrade` PowerShell cmdlet to  upgrade an existing ServiceControl instance to version 4.
+
+```ps
+Invoke-ServiceControlInstanceUpgrade -Name <Instance to Upgrade>
+```
+
+If the ServiceControl instance being upgraded manages an audit queue, then additional parameters must be specified for the creation of a new ServiceControl Audit instance.
+
+```ps
+Invoke-ServiceControlInstanceUpgrade `
+  -Name <Name of main instance> `
+  -InstallPath <Path for Audit instance binaries> `
+  -DBPath <Path for the Audit instance database> `
+  -LogPath <Path for the Audit instance logs> `
+  -Port <Port for the Audit instance api> `
+  -DatabaseMaintenancePort <Port for the Audit instance embedded database> `
+  [-ServiceAccountPassword <password for service account>] `
+  [-Force]
+```
+The following information is copied from the existing ServiceControl instance:
+
+- Audit queue
+- Audit log queue
+- Forward audit messages
+- Audit retention period
+- Transport
+- Connection string
+- Host name
+- Service account
+
+NOTE: If this instance uses a domain account, the the account password must be supplied.
+
+The name of the new audit instance will be derived from the name of the original instance.
+
+
+## Upgrading multi-instance deployments
+
+Upgrading a multi-instance ServiceControl deployment must be done in stages. Some stages may require the use of the powershell scripts. 
+
+### Upgrade the primary instance
+
+The first step is to upgrade the primary ServiceControl instance. If the primary instance has audit ingestion enabled, then a new ServiceControl Audit instance will be created for it.
+
+NOTE: Once the primary instance has been upgraded, it will not subscribe to events being published by new secondary instances. All subscriptions to existing secondary instances will be retained. As the primary instance no longer requires the transport address of the secondary instances to send subscription requests, the `Queue_Address` property has been dropped from the `ServiceControl/RemoteInstances` configuration setting.
+
+### Upgrade the secondary instances
```

Comment:
So I successfully tested the following upgrade procedure that doesn't require you to note down anything

```
$mainInstance = Get-ServiceControlInstances | Where-Object { $_.Name -eq "Particular.ServiceControl.0" }
$instanceToUpgrade = Get-ServiceControlInstances | Where-Object { $_.Name -eq "Particular.ServiceControl.1" }

Remove-ServiceControlInstance -Name $instanceToUpgrade.Name -RemoveLogs

New-ServiceControlAuditInstance -Name $instanceToUpgrade.Name `
 -Port $instanceToUpgrade.Port `
 -DatabaseMaintenancePort $instanceToUpgrade.DatabaseMaintenancePort `
 -DBPath $instanceToUpgrade.DBPath `
 -InstallPath $instanceToUpgrade.InstallPath `
 -LogPath $instanceToUpgrade.LogPath `
 -AuditQueue $instanceToUpgrade.AuditQueue `
 -AuditLogQueue $instanceToUpgrade.AuditLogQueue `
 -AuditRetentionPeriod $instanceToUpgrade.AuditRetentionPeriod `
 -ConnectionString $instanceToUpgrade.ConnectionString `
 -Transport $instanceToUpgrade.TransportPackageName `
 -ServiceControlQueueAddress $mainInstance.Name `
 -Force
```

I wasn't able to assign the ForwardAuditMessages property for some reason but the rest worked. Maybe helpful to add?